### PR TITLE
Add E2E data-attributes to Medical Copays key elements.

### PR DESCRIPTION
**Pull request by tlei123**
## Description
Add `data-e2e-id` HTML-attributes to key elements in Medical Copays app, to facilitate upcoming automated E2E-testing.

## Original issue(s)
n/a

## Testing done
- No app-functionalities are affected by these HTML-attributes.
- No Cypress specs created yet to reference these attributes.

## Acceptance criteria
- [ ] Reviewer(s) approved changes.

## Definition of done
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by tlei123** [comment]
@LinnJS , please review and approve the `data-e2e-id` HTML-attributes I added to make upcoming E2E testing easier.  Thanks much!

File: src/applications/medical-copays/components/Alerts.jsx [hunk 1]
```diff
@@ -8,7 +8,11 @@ import Telephone, {
 const Alert = ({ children }) => children;
 
 Alert.Error = () => (
-  <va-alert class="row vads-u-margin-bottom--5" status="error">
+  <va-alert
+    class="row vads-u-margin-bottom--5"
+    status="error"
+    data-e2e-id="error-alert"
```
Comment (craigwheeler) [inline src/applications/medical-copays/components/Alerts.jsx:14]: hey @tlei123 👋  .. noticed were using `data-e2e-id` here but the [docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/Cypress-Best-Practices-on-VA.gov-Platform.1870430211.html#CypressBestPracticesonVA.govPlatform-data-testidAttribute) recommend using `data-testid` so wanna confirm which test id we should be using going forward?

Comment (tlei123) [reply src/applications/medical-copays/components/Alerts.jsx:14]: good catch, @craigwheeler !  i'll update.  didn't know Platform's decided on a standard.

**Review by craigwheeler — APPROVED** [review]
LGTM! 🚀

**Pull request merged**